### ISSUE TITLE
Angular-stripe needed?

### ISSUE THREAD

**Issue by jrmcdona** [noodlio/noodlio-pay-ionic-example#3]
Hello!  So we need to install Angular-stripe?  The docs don't make that very clear.   I can see it in the example source code.    Fairly certain angular-stripe doesn't work with the latest version of ionic.

Suggestions?

**Comment by archive-a1** [comment]
I haven't tested it yet with the latest version of ionic. You should be able to install angular-stripe through the npm command `bower install angular-stripe`.

As for alternatives, i.e. if you want to avoid using Stripe Checkout, then you can build your own form and retrieve the stripeToken by sending a HTTPS request to the route [/tokens/create](https://market.mashape.com/noodlio/noodlio-pay-smooth-payments-with-stripe#tokens-create). 

So in short: no, you don't need angular-stripe to use the Noodlio Pay server.
